magicctapipe/scripts/lst1_magic/coincident_brokenGPS.py

```diff
@@ -0,0 +1,418 @@
+#!/usr/bin/env python
```

Comment:
you could add the two new scripts to setup.cfg which will allow to run them as executables without explicitely invoking python